notebooks/250-fastcomposer-image-generation/250-fastcomposer-image-generation.ipynb

```diff
@@ -0,0 +1,1352 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    !pip install -q transformers==4.25.1 accelerate datasets evaluate diffusers==0.16.1 xformers triton scipy clip gradio
</pre><p>do we really need strict transformersand diffusers versions?</p><p>As I know, now recommendation to have transformers &gt;= 4.30.1 (due to security vulnerability)</p><p> </p><p>xformers and triton are nvidia GPU specific packages, they are should be not necessary if you use torch CPU (as in our base requirements, you can look in 121 notebook to see how to specify its install in notebook)</p><p>also please add quitas for requirements with version specifications</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
This requirement string from requirements.txt of fastcomposer. I removed unnecessary requirements and pins, but unfortunately fastcomposer uses old transformers API that was changed in 4.30.1. So, 4.29.2 - last compatible version.